### PR TITLE
Google translate

### PR DESCRIPTION
**Pull request by all1124**

### CODE REVIEW

**Comment by all1124** [comment]
looks like there was a duplicate. i updated the file.

**Comment by NadyaIlinskiy** [comment]
@all1124  - conflicts have to be resolved before merge approval: 
Conflicting files
package-lock.json
package.json 
Also - if we're not using translate file anymore - it should be removed (it can be preserved locally if needed).

**Review by james-ad — APPROVED** [review]
All changes look good. Thanks for adding comments!

**Pull request merged**
